gax/operation.h

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAPIC_GENERATOR_CPP_GAX_OPERATION_H_
+#define GAPIC_GENERATOR_CPP_GAX_OPERATION_H_
+
+#include "google/longrunning/operations.pb.h"
+#include "gax/call_context.h"
+#include "gax/operations_stub.h"
+#include "gax/status.h"
+#include "gax/status_or.h"
+#include <memory>
+
+namespace google {
+namespace gax {
+
+/**
+ * Operation is a user-visible primitive used to support Long Running Operations
```

Comment:
Should you break that  one-line description to its own paragraph so Doxygen can automatically make it into the "brief" description?

Reply:
Probably. Never written for Doxygen before, so there's lots of stuff that's off and I don't know about it.